include/drivers/st25r3911b_nfc.rst

```diff
@@ -0,0 +1,88 @@
+.. _st25r3911b_nfc_readme:
+
+NFC Reader ST25R3911B
+#####################
+
+The NFC Device ST25R3911B can be used to read and write NFC-A compatible tags (tags that support the ISO/IEC 14443 standard), creating an NFC polling device.
+
+This library is intended to be used with the NFC Reader ST25R3911B Nucleo expansion board (X-NUCLEO-NFC05A1).
+
+NFC Reader hardware
+*******************
+
+The X-NUCLEO-NFC05A1 is an Arduino-compatible shield that can be used to communicate over NFC.
+It contains the NFC Reader chip ST25R3911B.
+
+The ST25R3911B can both generate its own field to read and write NFC tags and behave like a tag itself.
+It is a slave device, and the external microcontroller initiates all communication.
+
+Communication is performed by 4-wire Serial Pheripheral Interface (SPI).
+The ST25R3911B sends an interrupt request (pin IRQ) to the microcontroller.
+For the documentation of the NFC chip, see the `ST25R3911B chip documentation`_.
+
+Functionality
+*************
+
+The library provides the following functionality:
+
+* Detecting ISO/IEC 14443 A tags and reading their UID
+* Performing automatic collision resolution
+* Data exchange with the tag (write/read) with manually set Frame Delay Time
+
+  * With auto-generated CRC
+  * With manually generated CRC
+
+* Putting the tag to sleep
+
+The library currently supports only NFC-A technology.
+
+Architecture
+************
+
+The library consists of four layers:
+
+* Layer responsible for SPI communication: :ref:`st25r3911b_nfc_spi`
+* Layer to manage ST25R3911B interrupts: :ref:`st25r3911b_interrupts`
+* Layer with common reader functionality that can be shared by all NFC technologies: :ref:`st25r3911b_common`
+* NFC-A layer, which is responsible for NFC-A technology activities: :ref:`st25r3911b_nfca`
+
+The library requires calling its specific function periodically.
+
+API documentation
+*****************
+
+.. _st25r3911b_nfc_spi:
+
+ST25R3911B NFC SPI
```

Comment:
I am confused. I thought this API (as well as `_interrupts` and `_common`) was supposed to be private for this library, and only  `_nfca` was to be public.

Reply:
Yes, no needed.